lib/created_message_queue_handler.ts

```diff
@@ -125,16 +125,22 @@ export async function handleMessage(
 
   if (isMessageStorageEnabled) {
     // if the recipient wants to store the messages
-    // we add the content of the message to the message record
-    const errorOrUpdatedMessage = await messageModel.update(
+    // we add the content of the message to the blob storage for later retrieval
+    const errorOrRetrievedMessage = await messageModel.attachStoredContent(
       retrievedMessage.id,
       retrievedMessage.fiscalCode,
-      m => {
-        return { ...m, content: messageContent };
+      {
```

Comment:
just storing `messageContent` should be sufficient?

i.e. a `Message` attachment ===  `MessageContent`

Reply:
it may be sufficient but there's a case for storing the whole message (fiscalCode etc.): if we will implement an azure elasticsearch task running on the whole archive, it would be better to have all the information in the same blob. 

otherwise, we can just store the message content.